.buildkite/pipeline.yml

```diff
@@ -138,6 +138,7 @@ steps:
       TMPDIR: "/cache"
 
   - block: 'Run benchmark (api)'
+    if: '(build.branch !~ /^gh-readonly-queue\/master/) && (build.branch != "master") && (build.branch !~ /^release-candidate/)'
```

Comment:
This condition does exactly the opposite to what is written in the commit name
"Unblock Benchmark API steps on master and release branches":

if: 'not on the merge queue && not on master && not on a release candidate branch'